### ISSUE TITLE
Guitar ties don't play as connected note

### ISSUE THREAD

**Issue by Aliglider** [musescore/MuseScore#16968]
### Issue type

General playback bug

### Bug description

When two notes are tied on electric guitar they sound normal, but when changed to overdrive or distortion using text they play as two individual notes

### Steps to reproduce

Tie two notes on overdrive or distortion guitar

### MuseScore Version

4.02

### Regression

Yes, this used to work in Musescore 3.x and now is broken

### Operating system

Windows 10

**Comment by Jojo-Schmitz** [comment]
Sample scores in https://musescore.org/en/node/344207 and https://musescore.org/en/node/347179

**Comment by DmitryArefiev** [comment]
@bkunda Here how it looks:


https://github.com/musescore/MuseScore/assets/90187801/6b4145b9-f472-4787-89e0-9125edb5dc21

**Comment by atirut-w** [comment]
Why is such a major bug not fixed yet for this long? This bug has been there since the release of 4.0 and it affects me because I do guitar transcriptions among other things.

**Comment by cadiz2** [comment]
Not only guitar, see: https://musescore.org/en/node/350825

**Comment by cadiz2** [comment]
Reported again: https://musescore.org/en/node/346430

**Comment by DmitryArefiev** [comment]
Yes, already was logged separately #21195, and will be fixed soon (in 4.3)
